integration_tests/sqs_consumer_tests.py

```diff
@@ -0,0 +1,85 @@
+import os
+import time
+from datetime import datetime
+
+from molange.consumers import MessageConsumer
+from molange.drivers import get_sqs_driver, Message
+from molange.event_manager import event_manager
+from molange.publisher import get_sns_message_publisher
+from molange.subscribers import Subscriber
+
+
+# events.py
+class FollowCreatedEvent:
```

Comment:
dataclass pels events estaria pepi, no?

Reply:
ho vaig pensar pero aleshores estem forçats a tenir python3.7 no?